src/static/css/pages/beta.less

```diff
@@ -77,6 +72,7 @@
     .respond-to-min(801px, {
         // At this width, the wrapper gains `position: relative`,
         // so the button position has to be adjusted accordingly.
+        // This number depends on something in cf-layout
```

Comment:
So I just learned while I was working on this that `.respond-to-min` does that for us. https://github.com/cfpb/cf-core/blob/f9e72f8480289e4678119f2fef364d5a3c094c63/src/cf-media-queries.less#L70